### PR TITLE
dist/tests/philip: Removed old code and tests

### PR DESCRIPTION
**Pull request by MrKevinWeiss**
### Contribution description
Removed unneeded and unused tests and code.  Since PHiLIP is now released and hosted on the RAPstore server and automated HiL can be run in the RobotFW-tests we should clean out the unused tests and library.

### Testing procedure
Just murdock.  You can also search the RIOT directory for any signs of the philip interface or tests.  There should be none

### CODE REVIEW

**Comment by MrKevinWeiss** [comment]
I don't think it needs to be deprecated but that is for the community to decide.  I don't know of anyone aside from myself and @yegorich who may have used this.

**Comment by MrKevinWeiss** [comment]
Thanks, I will address!

**Comment by MrKevinWeiss** [comment]
Or should I remove the whole directory (maybe it messes around with the test discovery?)

File: tests/periph_i2c/tests/README.md [hunk 1]
```diff
@@ -1,35 +1,3 @@
...
-
-Notes
-==========
-- If no serial connection to the BPT the test should still be able to run, it just will not be able to reset.
-- Connect the SDA, SCL and if possible the DUT reset pin.
-- Autoconnect *may* work if no ports are specified.
-- Default baud rate is 115200
+Automated tests can be found in the
+[RobotFW-tests](https://github.com/RIOT-OS/RobotFW-tests/tree/master/tests/periph_i2c)
+area
```
Comment (smlng) [inline tests/periph_i2c/tests/README.md]: nit-picky: /area/repository/ ?

File: tests/periph_uart/README.md [hunk 2]
```diff
@@ -1,3 +1,3 @@
-Automated tests can be found in the
+_noteAutomated tests can be found in the
```
Comment (smlng) [inline tests/periph_uart/README.md]: add colon+space after `note` see i2c readme

File: tests/periph_i2c/README.md [hunk 3]
```diff
@@ -1,15 +1,24 @@
 Expected result
 ===============
-This test enables you to test all available low-level I2C functions using the shell.  Consult the 'help' shell command for available actions.
+This test enables you to test all available low-level I2C functions using the shell.
+Consult the 'help' shell command for available actions.
```
Comment (smlng) [inline tests/periph_i2c/README.md]: wrong single quotes need to use ` not '

Comment (MrKevinWeiss) [reply tests/periph_i2c/README.md]: Yup thanks!

File: tests/periph_uart/README.md (hunk 2 above)
Comment (MrKevinWeiss) [reply tests/periph_uart/README.md]: doh :confounded:

**Comment by MrKevinWeiss** [comment]
@smlng Have your comments been addressed and can I squash?

**Comment by smlng** [comment]
please squash

**Comment by MrKevinWeiss** [comment]
Many thanks @smlng!

**Pull request merged**